cf_xarray/tests/test_helpers.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+from numpy.testing import assert_array_equal
+
+import cf_xarray as cf  # noqa
```

Comment:
```suggestion
import cf_xarray as cfxr  # noqa
```

I think we should encourage this version.